application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -476,6 +476,12 @@
 #xwikicontent {
   overflow: visible;
 }
+
+#xpollTableWrapper{
+  width:100%; 
+  overflow-x:scroll;
```

Comment:
Doesn't this force the horizontal scroll bar even when not needed?

Reply:
Yes, but it's not functional when the table fits the page.
![image](https://user-images.githubusercontent.com/56109799/198546363-de7989e7-26cd-4945-a131-d42b54ec21fc.png)

Reply:
I can leave `overflow-x: auto` as an option, and it will disappear when the table fits.